roxie/ccd/ccdserver.cpp

```diff
@@ -26354,6 +26355,8 @@ class CRoxieServerKeyedJoinBase : public CRoxieServerActivity, implements IRecor
         indexReadInput = NULL;
         rootIndex = NULL;
         atmostsTriggered = 0;
+        unsigned allocatorFlags = roxiemem::RHFblocked;//roxiemem::RHFpacked; // MORE? unique? blocked?
```

Comment:
Should this be controlled via a hint? Or should you just make your mind up? Comment could use some cleanup at least...

Reply:
I agree.  I think blocked is sufficient - it has fewer side effects than packed, which would create a unique heap.  Assuming I am correct that alloc() is only called from a single thread...